src/runtime_src/xocl/core/debug.cpp

```diff
@@ -42,7 +42,6 @@ namespace {
 
 static bool s_debug_on = false;
 static std::string s_debug_log;
-static unsigned long s_zero = xocl::time_ns();
```

Comment:
Let's just do 
`s_zero = get_time_zero(); `

and get_time_zero() returns 0 by default but can be changed to original behavior under ini key if for some reason the old behavior is needed.

Reply:
Added get_time_zero function as requested.